<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEdit</class>
 <widget class="QDialog" name="NewEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edFamily">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
